yogstation/code/modules/uplink/uplink_item.dm

```diff
@@ -0,0 +1,9 @@
+/datum/uplink_item/stealthy_weapons/door_charge
+	name = "Explosive Airlock Charge"
+	desc = "A small, easily concealable device. It can be applied to an open airlock panel, booby-trapping it. \
+			The next person to use that airlock will trigger an explosion, knocking them down and destroying \
+			the airlock maintenance panel."
+	item = /obj/item/device/doorCharge
+	cost = 2
```

Comment:
Make the cost 0.1 because it's fucking shit and needs to die lol

Reply:
Salt much?